main/nvs_config.c

```diff
@@ -195,6 +195,84 @@ void nvs_config_set_float(const char *key, float value)
     nvs_config_set_string(key, str_value);
 }
 
+double nvs_config_get_double(const char * key, const double default_value)
```

Comment:
Use floats, which are stored as strings. That makes it easier to supply a value from a factory firmware. With blob, there's no mechanism in place to put this in the config.cvs files when doing a factory flash.